### PR TITLE
Adding Spheron CLI and IPFS service details on the docs

### PR DESCRIPTION
**Pull request by rekpero**
# Describe your changes

We've integrated references about Spheron's role in IPFS storage and website deployment on IPFS. Furthermore, we've enriched our content with details on the Spheron command line interface, including the steps to upload using the Spheron CLI.

Our platform facilitates easy deployment of static apps on IPFS, equipped with an inbuilt CI/CD for seamless automation. For a hands-on look, we've hosted an instance of this documentation forked to our organization. Feel free to explore it at [ipfs-docs-77ff9c.spheron.app](https://ipfs-docs-77ff9c.spheron.app/).

In addition, we conducted a swift lighthouse comparison with the [doc.ipfs.io](https://docs.ipfs.tech/) domain. The results highlight a significant performance enhancement in the instance deployed via Spheron.

Screenshot 2023-08-11 at 8 42 01 PM
Screenshot 2023-08-11 at 8 42 06 PM

We genuinely believe in the potential of the Spheron platform and encourage you to give it a try. Should you have any feedback or require modifications, please don't hesitate to inform us.

# Files changed

- docs/concepts/persistence.md
- docs/concepts/usage-ideas-examples.md
- docs/quickstart/publish_cli.md
- docs/welcome-builders.md

# What issue(s) does this address?

- Add spheron ipfs storage details 
- Add spheron command line interface to upload files to ipfs storage

# Does this update depend on any other PRs?

N.A

## Checklist before requesting a review
- [ ] Passing the beta version of the **Check Markdown links for modified files** check. Action results can be viewed [here](https://github.com/ipfs/ipfs-docs/actions/workflows/action.yml).

## Checklist before merging
- [ ] Passing all required checks (The beta **Check Markdown links for modified files** check is not required)

### CODE REVIEW

**Review by johnnymatthews — APPROVED** [review]
This looks good from my end, but I think @ElPaisano should weigh in here too before we get this merged.

File: docs/concepts/usage-ideas-examples.md
Comment (ElPaisano) [inline docs/concepts/usage-ideas-examples.md:157]:
```suggestion
[Spheron Network](https://spheron.network/) facilitates the seamless deployment of websites on IPFS and Filecoin. It offers developers the convenience of linking their websites or web applications hosted on platforms such as GitHub, Gitlab, and Bitbucket to the Spheron Network. Subsequently, any modifications made to a designated branch will trigger an automatic build and deployment on IPFS. In addition to this, Spheron incorporates integration features for the Ethereum Name Service (ENS) and other domain name services. The platform is poised to introduce an expanded array of deployment methods, including a command-line interface, API, and SDK.
```

**Review by ElPaisano — CHANGES_REQUESTED** [review]
This looks ok to me. However, could you please remove the spheron info added to publish_cli.md? We'd like to keep that tutorial scoped to web3.storage as it currently is.

**Comment by rekpero** [comment]
> This looks ok to me. However, could you please remove the spheron info added to publish_cli.md? We'd like to keep that tutorial scoped to web3.storage as it currently is.

Yes, I can remove it. Would it be possible to explore other places to highlight Spheron CLI? I believe it could greatly benefit the IPFS community to be aware of alternative CLIs like Spheron, especially in scenarios where services such as w3s might be unavailable. Your thoughts?

File: docs/how-to/peering-with-content-providers.md
Comment (lidel) [inline docs/how-to/peering-with-content-providers.md:121]: These Addresses are missing protocol  and port   (e.g. `/udp/4001/quic-v1` or `/tcp/4001`).
@rekpero are you able to find out what these are and add them in a new PR? :pray:

**Pull request merged**
